library/induction.red

```diff
@@ -1,85 +1,81 @@
-import bool
 import nat
 import path
 import unit
 
-def transport (A : type) (M N : A) (p : path A M N) (C : A → type) : C M → C N = 
-	λ cm → coe 0 1 cm in λ i →  C (p i)
+def transport (A : type) (M N : A) (p : path A M N) (C : A → type) : C M → C N =
+  λ cm → coe 0 1 cm in λ i → C (p i)
 
-def le : nat →  nat →  type = 
-	elim [
-		|	zero →  λ _ →  unit
-		| suc (m' → f) → 
-		elim [
-		| zero →  void
-		| suc n' →  f n'
-		]
-	]
+def le : nat → nat → type =
+  elim [
+  | zero → λ _ → unit
+  | suc (m' → f) →
+    elim [
+    | zero → void
+    | suc n' → f n'
+    ]
+  ]
 
-def le/suc (n: nat) : (m : nat) →  le n m → le (suc n) (suc m) =
-	elim n [
-	| zero →  \ _ _  →  triv
-	| suc n' →  \ m' l → l 
-	]
+def le/suc (n: nat) : (m : nat) → le n m → le (suc n) (suc m) =
+  elim n [
```

Comment:
A slicker way to write this it to put `(n : nat)` to the right of the colon, and change `elim n` to `elim`.